APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+
+* Agreeing upon the types of activities that can be funded,,
+* Establishing a process for deciding which specific activities should be funded,
+* Finding someone to do the work (if necessary),
+* Determining an appropriate compensation level.
+
+The following sections go into detail on each of these points.
+
+What to pay for
+^^^^^^^^^^^^^^^
+We can pay for anything which contributes to the broad mission of the Astropy Project. This includes, for example, code development, community engagement, education, promotion, etc. Funds can be used for many purposes, for example, paid developer time, teaching buyout, development of training materials, expenses for developer meetings or sprints, hosted hardware, and CI or other software services, carbon offsets, promotion and advertising, travel to conferences or meetings to promote Project activities.
+
+How to decide what to pay for
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Pay for things that are important and otherwise won’t get done. *Rationale: there is much more work to be done than there are funds to do all of that work. There are also many volunteers and externally sponsored people interested and willing to do some of the work without compensation from the Project.*
+2. Don’t rely on funding always being there. Plan for sustainability: how will activities be continued when the funds dry up? Invest in documentation, infrastructure, and easy to maintain workflows. For example, funded work on governance should always record the process clearly so that volunteers who follow have less work to do to figure it out. 
+3. Consider potential positive benefits to Astropy maintainers and contributors, maintainers of other projects, end users, and other stakeholders like packagers and educators. 
+4. Also consider the long-term costs to Astropy maintainers and contributors, to counterbalance 3.
+5. Do not increase the load on volunteer maintainers by assuming they are available to review or otherwise support the work. Instead, proposals should include paid time for reviews and other necessary activities. *Rationale: the effect of spending funds should be positive for everyone, not just for the people getting paid. This is also a matter of fairness.*
+
+Restricted / earmarked funding sources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Much of the Project’s funding comes from specific grants from private foundations and government agencies. In these cases, the fundable activities are explicitly described in the proposal text. Such funds must be spent following the proposed plan and often come with further restrictions on who can be funded. For example, NASA funding can only go to individuals or institutions in the USA and EU funding is generally restricted to institutions or individuals in the EU or associated countries. The Finance Committee should make these restrictions clear to the community so that they may submit funding requests aligned with the existing grants.
```

Comment:
Is it true that NASA funding can only go to folks in the US? I thought it was possible to sub-contract externally to the US with NASA money?

Reply:
There are two points in here:
- NASA money has some restrictions but it's not absolute. Yes, you can use NASA money to sub-contract outside of the US (e.g. purchase mirrors or CCDs for your space mission from companies outside of the US), but you can't sub-grant to non-US institutions for research (e.g. to pay for a Post-Doc to analyze your HST data). 
- For the purpose of this APE, we don't want to discuss the details of any individual grant. This is just to give an example to give the reader an idea what we are talking about.

I'll look how to re-word that.